source/DemoScene.cpp

```diff
@@ -80,5 +85,16 @@ bool DemoScene::update(Game &)
   if (Keyboard::getKeys()[OIS::KC_S]) {
     cameraNode->translate(0, 0, 10);
   }
+
+  if (Keyboard::getKeys()[OIS::KC_U]) {
+    illidan.stopAnimation();
+  }
+  if (Keyboard::getKeys()[OIS::KC_O]) {
+    illidan.setAnimation("Move", false);
+  }
+  if (Keyboard::getKeys()[OIS::KC_P]) {
+    illidan.setAnimation("Stand", false);
+  }
+  illidan.updateAnimations(fe.timeSinceLastFrame);
```

Comment:
Yes, which is in both case not visible for a human eye, and i don't think it's a problem for Ogre too.